internal/uvm/start.go

```diff
@@ -217,10 +263,21 @@ func (uvm *UtilityVM) Start(ctx context.Context) (err error) {
 	return nil
 }
 
+func (uvm *UtilityVM) StartClone(ctx context.Context) (err error) {
+	return uvm.startInternal(ctx, true)
+}
+
+// Start synchronously starts the utility VM.
+func (uvm *UtilityVM) Start(ctx context.Context) (err error) {
+	return uvm.startInternal(ctx, false)
+}
+
 // acceptAndClose accepts a connection and then closes a listener. If the
 // context becomes done or the utility VM terminates, the operation will be
 // cancelled (but the listener will still be closed).
 func (uvm *UtilityVM) acceptAndClose(ctx context.Context, l net.Listener) (net.Conn, error) {
+	ctx, span := trace.StartSpan(ctx, "uvm::acceptAndClose")
+	defer span.End()
```

Comment:
why this?

Reply:
This acceptAndClose phase is taking a long time when reconnecting with the GCS of a cloned UVM. Having this trace will be useful in future debugging sessions. Especially when we are concerned about the startup times of clones.